scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -123,6 +114,16 @@ def test_sort_vertices_of_regions_flattened(self):
         actual = list(itertools.chain(*sorted(sv.regions)))
         assert_array_equal(actual, expected)
 
+    def test_sort_vertices_of_regions_dimensionality(self):
+        points = np.array([[1, 0, 0, 0],
+                           [0, 1, 0, 0],
+                           [0, 0, 1, 0],
+                           [0, 0, 0, 1],
+                           [0.5, 0.5, 0.5, 0.5]])
+        with pytest.raises(TypeError, match="three-dimensional"):
```

Comment:
the context for the error is still larger than it needs to be--we only need to probe `sort_vertices_of_regions()` so that we don't accidentally catch an error from `SphericalVoronoi` itself. Anyway, that is pretty minor & could be cleaned up later if at all.